src/rcheevos.vcxproj

```diff
@@ -1,14 +1,46 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="DebugUnicode|Win32">
+      <Configuration>DebugUnicode</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
```

Comment:
it doesn't make sense to have Unicode builds of this project. the solution configuration should just reference the Debug/Release configurations of this project.

Reply:
sure, for the dropdown I'll get rid of the Unicode. I guess devs will have to turn it on in the configuration manager if they want to build a Unicode version. The Unicode builds where there just not in the dropdown near the task bar.